assets/js/blocks/attribute-filter/block.tsx

```diff
@@ -582,20 +582,47 @@ const AttributeFilterBlock = ( {
 				className={ `wc-block-attribute-filter style-${ blockAttributes.displayStyle }` }
 			>
 				{ blockAttributes.displayStyle === 'dropdown' ? (
-					<DropdownSelector
-						attributeLabel={ attributeObject.label }
-						checked={ checked }
+					<div
 						className={ classNames(
-							'wc-block-attribute-filter-dropdown',
+							'wc-blocks-components-form-token-field-wrapper',
```

Comment:
How do you feel about having a separate component what handles all the styling and default props needed for the FormTokenField before we can use it?

This was the customizations below would only have to be applied once, and it can be then included in other places in the repo.

With a new component we can also add a story so that we can showcase it in Storybook